Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -798,6 +914,54 @@ class AppStoreConnectService {
             .await()
     }
 
+    func pullTestFlightConfigs() throws -> [TestFlightConfiguration] {
+        let apps = try listApps(bundleIds: [], names: [], skus: [], limit: nil)
```

Comment:
We will need to support pulling a subset of apps (filtered by bundleIds probably).